### PR TITLE
Django 2.2 & 3.0

### PR DESCRIPTION
**Pull request by browniebroke**
- [x] Add Django 2.2 and 3.0 to the build matrix
- [x] Add Python 3.7 & 3.8
- [x] Fix issues

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -63,7 +65,7 @@ def read(*names, **kwargs):
     url='https://github.com/nedbat/django_coverage_plugin',
     packages=['django_coverage_plugin'],
     install_requires=[
-        'coverage >= 4.0',
+        'coverage >= 4.0 , < 5',
```
Comment (nedbat) [inline setup.py]: Why do this?

Comment (jambonrose) [reply setup.py]: From the test output of the first commit:
```pytb
Traceback (most recent call last):
  File "tests/test_extends.py", line 16, in test_empty_block
    text = self.run_django_coverage()
  File "tests/plugin_test.py", line 139, in run_django_coverage
    for pl in self.cov.plugins:
AttributeError: 'Coverage' object has no attribute 'plugins'
```

I've been planning to switch to [coverage.CoveragePlugin](https://coverage.readthedocs.io/en/coverage-5.0.3/api_plugin.html#coverage.CoveragePlugin), but I've been swamped at work.

Comment (browniebroke) [reply setup.py]: Yes, was failing without. I feel like it's out of scope of this PR?

Comment (jambonrose) [reply setup.py]: I'm in agreement.

Comment (nedbat) [reply setup.py]: OK: #66

**Comment by browniebroke** [comment]
@jambonrose I think I made all the changes you requested. Let me know if you see anything else.

File: setup.py (hunk 1 above)
Comment (browniebroke) [reply setup.py]: Given that this is broken on master (I think), should I move this to a separate pull request so you can make a patch release of this plugin before adding more stuff?

**Pull request merged**